op-service/signer/cli_test.go

```diff
@@ -72,6 +72,13 @@ func TestInvalidConfig(t *testing.T) {
 				config.TLSConfig.Enabled = true
 			},
 		},
+		{
+			name:     "MissingAddress",
+			expected: "signer endpoint and address must both be set or not set",
+			configChange: func(config *CLIConfig) {
+				config.Endpoint = "http://localhost"
```

Comment:
Doesn't this need a port? At least `:0` to be explicit for random port selection? I guess it does not reach the endpoint parsing part, because of the error on the other missing attribute?

Reply:
Yep, the code wasn't reaching the endpoint parsing part. Also, this test case was duplicative, so I removed it :)